lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -126,6 +126,16 @@ void _hideAutofillElements(DomHTMLElement domElement,
       ..left = '${offScreenOffset}px';
   }
 
+  if (shouldHideElement) {
+    elementStyle
+      ..width = '0'
+      ..height = '0';
+  }
+
+  if (shouldDisablePointerEvents) {
+    elementStyle.pointerEvents = 'none';
```

Comment:
This seems like a good thing to do in all browsers, what do you think?

Reply:
We probably could but I noticed one side effect if we do this, where the dev tools inspector can never directly inspect any of the inputs.  It's a minor annoyance, but since setting `pointer-events` to `none` doesn't seem to have an effect on browsers outside of Safari, I decided to keep the change exclusive to Safari.